frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -134,7 +134,7 @@ const deleteUserRoleAssignment = async (
     try {
         userRoleAssignments.value = await deleteAndRefreshUserRoleAssignments(
             assignment.user_role_xref_id,
-            assignment.role.application_id
+            assignment.role.application.application_id
```

Comment:
Hi Ian, I'm still wondering if we need to include the "application", cause the role table already has the application_id information... <img width="276" alt="Screen Shot 2024-08-13 at 1 04 39 PM" src="https://github.com/user-attachments/assets/f1f457e0-6dfd-4649-bd2f-044ed7252f91">

Reply:
Initially it had the application id; but the schema or the return to frontend does not necessary (better if not in some cases) need to match exact from the model. I only changed it to grab the application_name for convenience; we could include both but I think it is not necessary.

Reply:
oh I see... or we could pass the application_name as a parameter? so it won't impact the existing schema, and will not include additional application information in the return to frontend?

<img width="672" alt="image" src="https://github.com/user-attachments/assets/f534063b-f553-494e-b747-8ee6539442c6">

Reply:
I would not take application_name from audit_event_log's property; either getting from the main event response or from db again (which is what schema change to take advantage of ORM model conversion to schema).

It's been a long time not refactoring this, but this audit_log can actually be a stand-alone call rather than in router; and router may not necessary know audit_log has such property.

Reply:
sure, then I'll follow the same for admin management so it's consistent, thanks!